helm/cilium/templates/extra-policies/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if or .Values.extraPolicies.allowEgressToCoreDNS.enabled .Values.extraPolicies.allowEgressToProxy.enabled -}}
```

Comment:
We anyhow need to adapt all apps that are installed on vintage, which I presume includes `cilium-app`. So that change makes sense but should be in scope of another issue.